### PR TITLE
Fetching the data from Input.txt File

### PR DESCRIPTION
**Pull request by PRABHHHH**
Worked with Input file

### CODE REVIEW

File: Operational/Instructions.robot [hunk 1]
```diff
@@ -1,6 +1,9 @@
 *** Settings ***
 Library  SeleniumLibrary
 Library  String
+Library  Dialogs
```
Comment (vineethravi909) [inline Operational/Instructions.robot]: Please remove unwanted libraries

File: Operational/Instructions.robot [hunk 2]
```diff
@@ -31,23 +34,47 @@ Then Users Checks For Recently Added Item
 
 
 User Searches For An Item And Clicks On It
+    ${TextFile} =  Get File  Input.txt
+    ${List} =  Split to lines  ${TextFile}
+    ${Stringg} =  Convert To String  ${TextFile}
+
```
Comment (vineethravi909) [inline Operational/Instructions.robot]: Why is there an extra line?
Any reason?

File: Operational/Instructions.robot [hunk 3]
```diff
@@ -31,23 +34,47 @@ Then Users Checks For Recently Added Item
...
+    Log To Console  ${InputMeasurements}
+    Set Suite Variable    ${Param}
+    Set Suite Variable    ${InputMeasurements}
     Input Text  ${Searchbar}  ${Shirt}
     Click Element  ${SearchButton}
     Click Element  ${Prod}
 
 
 
 Then Opens Size Chart
```
Comment (vineethravi909) [inline Operational/Instructions.robot:49]: Why is the argument removed?

File: Operational/Instructions.robot [hunk 4]
```diff
@@ -31,23 +34,47 @@ Then Users Checks For Recently Added Item
...
-    [Arguments]  ${ARGS}
     Switch Window  New
     Click Element  ${Chart}
     ${rowCount}=    Get Element Count    ${Rows}
     ${colCount}=    Get Element Count    ${Columns}
     Log To Console  ${colCount}
-    Log To Console  ${rowCount}
+    Log To Console  ${rowCount} 
+
```
Comment (vineethravi909) [inline Operational/Instructions.robot]: Too many blank lines

File: Tests/TestCases.robot [hunk 5]
```diff
@@ -22,13 +23,13 @@ User Should Be Able To Add Another Item In The Cart
     Then Users Checks For Recently Added Item  ${any} 
 
 User Should Be Able To Add A Product Into The Cart Based On Any Parameter In Size Chart
-    [Tags]  First
+    [Tags]  Second
```
Comment (vineethravi909) [inline Tests/TestCases.robot]: Give appropriate Tag names

**Review by vineethravi909 — CHANGES_REQUESTED** [review]
Please resolve the comments

File: Operational/Instructions.robot (hunk 1 above)
Comment (PRABHHHH) [reply Operational/Instructions.robot]: Resolved

File: Operational/Instructions.robot (hunk 3 above)
Comment (PRABHHHH) [reply Operational/Instructions.robot:49]: Was trying Diff things

**Review by vineethravi909 — APPROVED** [review]
LGTM

**Pull request merged**